src/t8_geometry/t8_geometry_implementations/t8_geometry_cad.cxx

```diff
@@ -921,6 +915,220 @@ t8_geometry_cad::t8_geom_evaluate_cad_hex (t8_cmesh_t cmesh, t8_gloidx_t gtreeid
   T8_FREE (interpolated_coords);
 }
 
+void
+t8_geometry_cad::t8_geom_evaluate_cad_prism (t8_cmesh_t cmesh, t8_gloidx_t gtreeid, const double *ref_coords,
+                                             const size_t num_coords, double *out_coords) const
+{
+  T8_ASSERT (active_tree_class == T8_ECLASS_PRISM);
+
+  /* Compute coordinates in global space from ref_coords in order to shift them afterwards */
+  t8_geom_compute_linear_geometry (active_tree_class, active_tree_vertices, ref_coords, num_coords, out_coords);
+
+  const t8_locidx_t ltreeid = t8_cmesh_get_local_id (cmesh, gtreeid);
+  const int num_edges = t8_eclass_num_edges[active_tree_class];
```

Comment:
I think  it would be better to add a Makro to the prism-implementation, because the number of edges is not something that changes and the `active_tree_class` is the same during this function. You could then also use the Makro directly later in the code instead of the variable.